docs/fsharp/tour.md

```diff
@@ -1,4 +1,158 @@
 # Tour of F# #
 
+The best way to learn about F# is to read and write F# code.  This tour of some of the features of the language will highlight some things and give you some code snippets that you can execute on your machine.  To learn about setting up a development environment, check out [Getting Started](tutorials/getting-started/index.md).
+
+There are two primary concepts in F#: functions and types.  This tour will emphasize features of the language which fall into these two concepts.
+
+## How to Run the Code Samples
+
+>![NOTE]
+Eventually, this documentation site will have the ability to run these code samples directly in your browser.
+
+The quickest way to run these code samples is to use [F# Interactive](tutorials/fsharp-interactive/index.md).  Just copy/paste the code samples and run them there.  Alternatively you can set up a project to compile and run the code as a Console Application in [Visual Studio](tutorials/getting-started/getting-started-visual-studio.md) or [Visual Studio Code and Ionide](tutorials/getting-started/getting-started-vscode.md).
+
+## Functions and Modules
+
+The most fundamental pieces of any F# program are Functions organized into Modules.  [Functions](language-reference/functions/index.md) perform work on inputs to produce outputs, and they are organized under Modules, which are the primary way you group things in F#.  They are defined using the [`let` binding](language-reference/functions/let-bindings.md), which give the function a name and define its arguments.
```

Comment:
When I introduce a new term, I've been following the convention of bold italic, instead of capitalization. Here, you introduce ***functions*** and ***modules***.

I like this style because I can also use the code font, and use all lower case when it is a keyword, and case matters.

/cc @mairaw